releases/v20.1.0-rc.1.md

```diff
@@ -0,0 +1,276 @@
+---
+title: What&#39;s New in v20.1.0-rc.1
+toc: true
+summary: Additions and changes in CockroachDB version v20.1.0-rc.1 since version v20.1.0-beta.4
+---
+
+## April 13, 2020
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.1.0-rc.1.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.1.0-rc.1.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.1.0-rc.1.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.1.0-rc.1.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v20.1.0-rc.1
+~~~
+
+### Security updates
+
+- The new `--unencrypted-localhost-http` flag for [`cockroach start`](../v20.1/cockroach-start.html) and [`cockroach start-single-node`](../v20.1/cockroach-start-single-node.html) forces the HTTP listener to bind to `localhost` addresses only and disables the TLS protocol. In secure clusters, this makes the Admin UI reachable with an `http://` URL without requiring certificate or CA setup. [#46472][#46472] {% comment %}doc{% endcomment %}
+
+### General changes
+
+- Transactions reading a lot of data behave better when exceeding the memory limit set by the `kv.transaction.max_refresh_spans_bytes` [cluster setting](../v20.1/cluster-settings.html). Such transactions now attempt to resolve the conflicts they run into instead of being forced to always retry. Increasing `kv.transaction.max_refresh_spans_bytes` should no longer be necessary for most workloads. [#46803][#46803]
+- Schema changes started in v19.2 before [upgrading to v20.1](../v20.1/upgrade-cockroach-version.html) are now now automatically migrated in the background after the upgrade is finalized so that they can run to completion. [#47073][#47073] {% comment %}doc{% endcomment %}
```

Comment:
~Did I remember that in another PR we are saying not to do an upgrade while schema changes are running?  I think it's your upgrade finalization PR (?).